service/posts/views.py

```diff
@@ -11,20 +11,12 @@ class PublicPostsList(generics.ListAPIView):
     pagination_class = PostsPagination
     serializer_class = PostSerializer
     authentication_classes = (NodeBasicAuthentication,)
-    permission_classes = (IsAuthenticated,)
+    # No permission class
 
     def get_queryset(self):
-        node = self.request.user
-
-        if not node.share_posts:
-            return Post.objects.none()
-
-        queryset = Post.objects.filter(visibility="PUBLIC")
+        remote_node = self.request.user
```

Comment:
I might be missing something but how is it that `self.request.user` relates to our `Node` model? Is it being related through our `Author` user profile by Django?